### PR TITLE
first index file

### PR DESCRIPTION
**Pull request by Ann0426**
hey try this out

### CODE REVIEW

**Review by YaoChungLiang — COMMENTED** [review]
Wait a minute, what is babel.js for? Sounds like irrelevant.

File: index.html [hunk 1]
```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+
+<html>
+<head>
+<meta charset="UTF-8" />
+<title>Hello React!</title>
+<script src="https://cdn.staticfile.org/react/16.4.0/umd/react.development.js"></script>
+<script src="https://cdn.staticfile.org/react-dom/16.4.0/umd/react-dom.development.js"></script>
+<script src="https://cdn.staticfile.org/babel-standalone/6.26.0/babel.min.js"></script>
+</head>
```
Comment (Ann0426) [inline index.html]: idk

**Review by Ann0426 — COMMENTED** [review]
lets call it a day

**Comment by Ann0426** [comment]
123456

File: index.html (hunk 1 above)
Comment (YaoChungLiang) [reply index.html]:
```suggestion
<script src="https://cdn.staticfile.org/react/16.4.0/umd/react.development.js"></script>
<script src="https://cdn.staticfile.org/react-dom/16.4.0/umd/react-dom.development.js"></script>
</head>
```

**Pull request merged**